templates/bare/package.json

```diff
@@ -26,10 +26,12 @@
   },
   "devDependencies": {
     "@aragon/test-helpers": "^2.0.0",
+    "eth-ens-namehash": "^2.0.8",
+    "eth-gas-reporter": "^0.1.1",
     "ganache-cli": "^6.0.3",
-    "solidity-coverage": "0.5.8",
     "solium": "^1.0.4",
     "truffle": "4.1.14",
+    "truffle-extract": "^1.2.1",
```

Comment:
Not sure we use this dependency (we don't extract the ABIs, and I'm not sure if we'll ever publish these templates to npm)

Reply:
We don't but it was in all the templates with their corresponding npm scripts, I didn't want to remove it in case it was useful for someone